src/index.tsx

```diff
@@ -1,15 +1,17 @@
-import * as React from 'react';
-import * as codemirror from 'codemirror';
-import {Position} from "codemirror";
+import * as React from "react";
```

Comment:
Hey @andyrichardson - I'm going to help to get this merged if you're still up for it?
Soo... I don't see anything in the `.editorconfig` file that defines the spacing / quotes style for this project but your change introduces a number of purely stylistic changes that I think are ancillary. I personally like the style you're using but I don't want to mix/match. Would you be up for either a) removing stylistic changes b) committing changes to `.editorconfig` to either cause your style or the original one to be applied throughout?

Reply:
Hey @elsigh I'm more than happy to go with whatever stylistic preferences the project has! Sorry about the unnecessary changes, I have prettier on by default which goes HAM on everything.

Reply:
@andyrichardson - I had the same issue - I had to use Cmd-Shift-P to save wihout formatting.

Reply:
Gentle ping @andyrichardson =)

Reply:
I will sort this out this week!

I hope... 😉

Reply:
Would still love to merge this one if you get a little time

Reply:
Would love to see this merged still if possible! @andyrichardson